lib/routes/builds.js

```diff
@@ -1,12 +1,17 @@
 'use strict';
 
-const async = require('async');
-const url = require('url');
 const stringify = require('stringify-stream');
 const asynk = require('express-async-handler');
-const semver = require('semver');
+const path = require('path');
+const { tmpdir } = require('os');
+var Retryme = require('retryme');
 
 const stringifyOpts = { open: '[', close: ']' };
+var op = new Retryme({
```

Comment:
we actually need a single instance of this per thing being retried, otherwise they will be using the same retry state. This can be generated inline with `retry.op(opts).async()` where `retry` is `const retry = require('retryme');`